schemas/atlas/parcellationTerminologyVersion.schema.tpl.json

```diff
@@ -2,7 +2,6 @@
   "_type": "https://openminds.ebrains.eu/sands/ParcellationTerminologyVersion",
   "required": [
     "fullName",    
```

Comment:
"fullName" should be deleted; "shortName" should be kept.

Reply:
I made shortName required instead of fullName.